File: 1970-sorting-the-sentence/1970-sorting-the-sentence.go
func sortSentence(s string) string {
  words := strings.Split(s, " ")
    sort.Slice(words, func(i, j int) bool {
        return words[i][len(words[i])-1] < words[j][len(words[j])-1]
    })
    for i := range words {
        words[i] = words[i][:len(words[i])-1]
    }
    return strings.Join(words, " ")
}

File: 1970-sorting-the-sentence/1970-sorting-the-sentence.py
class Solution:
    def sortSentence(self, s: str) -> str:
        sentence=s.split()
        Sorted = sorted(range(len(sentence)), key=lambda i:sentence[i][-1])
        res=[sentence[j][:-1] for j in Sorted]   
        return ' '.join(res)    